internal/agent/httpjsonbatchsender/httpjsonbatchsender.go

```diff
@@ -0,0 +1,85 @@
+package httpjsonbatchsender
```

Comment:
```suggestion
package sender
```
кажется этого достаточно для понимания что делает пакет